src/core/action-manager.ts

```diff
@@ -31,9 +33,8 @@ class ActionManager {
     superuser: boolean,
     action: IAction | IExtension,
   }>;
-  
   private globals: {
-    [key: string]: Function
+    [key: string]: () => void, // Function,
```

Comment:
@shashaBot globalRegister callback don't need to return anything right?

Reply:
Since we cannot use Function as a type & I didn't know the what the callback signature is, I tried to define the format as () => void.
Should I instead define it simply as [key:string]: (...args: any[]) => void; or can you clarify what the callback type is?